src/sandstorm/web-session-bridge.c++

```diff
@@ -1315,6 +1316,13 @@ kj::Promise<void> WebSessionBridge::handleResponse(
       );
     }
 
+    // Set Referrer-Policy: same-origin. Otherwise, external sites linked
+    // from a grain can learn its randomized hostname from the Referer
+    // header. This knowledge could then potentially be used to launch an
+    // XSRF attack. We avoid using "no-referrer" here to prevent Chrome from
+    // sending "Origin: null" (see gateway.c++).
```

Comment:
I suspect if the code in `gateway.c++` ever gets updated, there's like a 98% chance we'll forget to update this comment. I'm not sure how much value the last sentence is really adding. If we're going to mention this wrinkle here, I'd prefer to word it in a way where it's not likely to bitrot.

Reply:
I've improved this such that the comment now references a commit id, which will not change.